common/v2/utils/web3.ts

```diff
@@ -14,61 +12,46 @@ interface Web3ProviderInfo {
   walletId: WalletId;
 }
 
-const WEB3_CONFIGS: {
```

Comment:
From what I see this object is only called within `config/wallets`.
Let's simplify and merge this into the config object.
That way we will also be able to merge  `getWeb3ProviderInfo` and `getWeb3Config`